src/routes/User.jsx

```diff
@@ -0,0 +1,95 @@
+// React Imputs
```

Comment:
Same comment as `AppRoutes`... is this new page/route needed? Does this not fit in `Forms`?

Reply:
The opposite, actually. Think we could move some forms in user instead like upload and delete documents.

Reply:
Ok, so when building the list of all the user's documents (let's just say it looks like the client list for an easy visual), that should go in `Users`?   This preview would happen onClick of a document.  Then there'd be other things from `forms` like search documents, delete document, etc ?

Reply:
I see the forms for upload, etc. on the User page itself rather than on the modal. The modal will just be for the file viewer.